core/settings.py

```diff
@@ -266,7 +264,7 @@
 EMAIL_HOST = env('EMAIL_HOST')
 EMAIL_HOST_USER = env('EMAIL_HOST_USER')
 EMAIL_HOST_PASSWORD = env('EMAIL_HOST_PASSWORD')
-EMAIL_PORT = env('EMAIL_PORT')
+EMAIL_PORT = 587
```

Comment:
Any specific reason why not to include port nr as variable?

Reply:
Port 587 should usually be the default for sending email with TLS. Most servers that we would use for sending mail will probably support 587 so I don't think it's worth adding extra options to the config that nobody will touch. It should be easy to add back later if needed.

Reply:
Yeah sure, probably won't have to use any of the other ports.